src/OrleansProviders/Streams/Common/Monitors/DefaultBlockPoolMonitor.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Providers.Streams.Common
+{
+    /// <summary>
```

Comment:
it is overridden by EventHubBlockPoolMonitor, with a different setting in LogProperties. Maybe we can reuse it in AzureQueueStreamProvider, etc in the future. It is intended to be overridden to provide different aggregation mechanism (LogProperties settings), since memoryStreamProvider and EHStreamProvider has different flavor on aggregation.